### PR TITLE
DOC: consistent doc-string with function declaration, added missing param doc-string for sql.py

### PR DESCRIPTION
**Pull request by terrytangyuan**
Added default values for params in doc-string and added missing param doc-string that are needed for new users.

### CODE REVIEW

File: pandas/io/sql.py [hunk 1]
```diff
@@ -1143,7 +1145,9 @@ def read_query(self, sql, index_col=None, coerce_float=True,
               to the keyword arguments of :func:`pandas.to_datetime`
               Especially useful with databases without native Datetime support,
               such as SQLite
-
+        chunksize : int, default None
+            If specified, return an iterator where `chunksize` is the number
+            of rows to include in each chunk.
         Returns
```
Comment (jorisvandenbossche) [inline pandas/io/sql.py]: needs a blank line above 'Returns'

**Comment by jorisvandenbossche** [comment]
small comment, otherwise looks good!

**Comment by terrytangyuan** [comment]
Done. Thanks! @jorisvandenbossche

**Comment by jorisvandenbossche** [comment]
Thanks!

**Pull request merged**